server/routes/users.js

```diff
@@ -52,7 +54,7 @@ const redirectProfile = (req, res, next) => {
 // @tested 	Yes
 router.get('/', (req, res) => {
   try {
-    res.status(200).render('home', {
+    res.status(200).render(path.join(__dirname, '../../client/views/home'), {
```

Comment:
those are not neccessary, we just have to tell ejs where the views are, itll get the correct ones then